modules/core/04-channel/keeper/packet.go

```diff
@@ -380,10 +380,10 @@ func (k Keeper) AcknowledgePacket(
 		)
 	}
 
-	if channel.State != types.OPEN {
+	if channel.State != types.OPEN && channel.FlushStatus != types.FLUSHING {
```

Comment:
how does this read to folks? Can refactor in a similar way to SendPacket but this conditional isn't as beefy.